packages/safe-tools-client/app/services/network.ts

```diff
@@ -0,0 +1,48 @@
+import Service from '@ember/service';
+
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+import {
+  getConstantByNetwork,
+  Networks,
+  networks,
+  supportedChainsArray,
+} from '@cardstack/cardpay-sdk';
+
+const DEFAULT_NETWORK = 'mainnet' as const; // TODO: add default based on env
+
+export default class Network extends Service {
```

Comment:
I suggest naming this class `NetworkService` to avoid conflicts with the Network type. I think that confusion has already crept in -- e.g. the argument type for the `onSelect` method below.